### DISCUSSION TITLE
Specify timeout per test case

### DISCUSSION THREAD

**Discussion by FabianOswald-WolfSystem** [ricaun-io/RevitTest#22] [Ideas]
Is there a way to specify a timeout per test case? I know that you can specify a timeout for the entire test project.

-> https://docs.nunit.org/articles/nunit/writing-tests/attributes/timeout.html

**Comment by ricaun** [comment]
For individual tests no, this attribute is not supported.

Is not possible to timeout a Revit method, if is running for too long you need to wait that to finish 😐

**Comment by FabianOswald-WolfSystem** [comment]
Okay, thanks for the answer, that makes sense.
